request_test.go

```diff
@@ -31,6 +32,7 @@ const examplePutRequest = "PUT /_matrix/federation/v1/send/1493385816575/ HTTP/1
 	" origin=\"localhost:8800\"" +
 	",key=\"ed25519:a_Obwu\"" +
 	",sig=\"+hmW6UjEXx7vMt2+MXO/EImSfdEYdBsZEOmpiz3evYktAgGNpGuNMBYXIA969WGubmceREKA/r1phasUFHBpDg\"" +
+	",destination=\"localhost:44033\"" +
```

Comment:
Are there some test cases with this field missing? Why add them here?

Reply:
These tests do an equality check apparently, so the value is needed. If it's missing, we ignore it and populate it ourselves with the `serverName`, so it should be fine.